src/api/internal/index.ts

```diff
@@ -141,7 +141,10 @@ export const getAccountBalance = async (
   } catch (e) {
     console.error(e);
   }
-  return response.balances[0];
+  return response.balances.filter(
+    // eslint-disable-next-line camelcase
+    (balance: { asset_code?: string }) => !balance.asset_code,
+  )[0];
```

Comment:
looks like if there isn't an `asset_code`, i should assume it's XLM. If that's wrong lmk!